### PR TITLE
Allow setting downscale replicas via namespace annotation

### PR DESCRIPTION
**Pull request by gordonbondon**
Sometimes it's useful to be able to set a number of downtime replicas for whole namespace

### CODE REVIEW

**Comment by hjacobs** [comment]
Thanks! Can you add a test for this case?

**Comment by gordonbondon** [comment]
@hjacobs done

File: tests/test_scaler.py [hunk 1]
```diff
@@ -298,3 +298,40 @@ def get(url, version, **kwargs):
...
+        elif url == 'deployments':
+            data = {'items': [
+                {
+                    'metadata': {
+                        'name': 'deploy-1', 'namespace': 'default', 'creationTimestamp': '2019-03-01T16:38:00Z',
+                    }, 'spec': {'replicas': 5}
+                },
+                ]}
+        elif url == 'namespaces/default':
+            data = {'metadata': {'annotations': {'downscaler/downtime-replicas': '3'}}}
```
Comment (hjacobs) [inline tests/test_scaler.py]: nit: you can re-use the `SCALE_TO` variable here (instead of "3")

Comment (gordonbondon) [reply tests/test_scaler.py]: fixed

**Comment by hjacobs** [comment]
Thanks again :smile:

**Pull request merged**